pkg/harvester/models/harvester/node.js

```diff
@@ -134,15 +134,12 @@ export default class HciNode extends HarvesterResource {
 
   get consoleUrl() {
     const url = this.metadata?.annotations?.[HCI_ANNOTATIONS.HOST_CONSOLE_URL];
+    const validator = /^[a-z]+:\/\//;
 
-    if (!url) {
+    if ((!url) || (url.match(validator) === null)) {
```

Comment:
This can be simplified as:
```suggestion
    if (!url?.match(validator)) {
```

Reply:
Thanks for the suggestion, sounds good.